zulip_bots/zulip_bots/lib.py

```diff
@@ -218,6 +246,16 @@ def handle_message(message):
                 return
 
         if is_private_message or is_mentioned:
+            # Handle any default commands first
+            if len(updated_defaults) > 0:
+                for command in updated_defaults:
+                    if command == '' and len(message['content']) == 0:
```

Comment:
Why would we need an extra if-clause for this case; shouldn't it be caught by the following if-clause?

Reply:
You mean that we could remove the first `if` block and the functionality would be covered by the second? That would seem to be the case, yes; possibly it is a remnant of when I was just covering the empty case.